integration-tests/src/test/java/tachyon/worker/DataServerIntegrationTest.java

```diff
@@ -107,6 +110,7 @@ private void assertValid(final DataServerMessage msg, final int expectedSize,
   @Before
   public final void before() throws IOException {
     System.setProperty(Constants.WORKER_DATA_SERVER, mDataServerClass);
+    System.setProperty(Constants.WORKER_NETTY_FILE_TRANSFER_TYPE, mNettyTransferType);
```

Comment:
This line only takes effect if the class is netty server, right?
Can we set the transfer type to be something like NOT USED for NIO server (in line 61)?